### PR TITLE
Chart popup

### PR DESCRIPTION
**Pull request by amosNash**
Implemented a popup to display a chart. 

I also removed client code that wasn't being used, and added some API calls to get the data needed for the chart.

### CODE REVIEW

File: client/src/components/itemColorOptions.jsx [hunk 1]
```diff
@@ -8,20 +7,13 @@ const ItemColorOptions = (props) => {
...
   const [color, setColor] = useState(props.color);
 
-  //handle box clicks which will take in productId and return productId
-
-  //pass productId up to app.jsx
   return (
     <div>
       <SectionDescriptionText>Color:</SectionDescriptionText> <b>{color}</b>
       <br></br>
       <ColorBoxBlack onClick={() => props.handleColorBoxClick(props.similarItems[0])} onMouseEnter={() => setColor('Black')} onMouseLeave={() => setColor(props.color)}></ColorBoxBlack>
```
Comment (bshresthap) [inline client/src/components/itemColorOptions.jsx:14]: Are these for color of product ?

File: client/src/components/ratingChart.jsx [hunk 2]
```diff
@@ -1,24 +1,135 @@
...
-        <HorizontalBar data={data}></HorizontalBar>
+        <Chart>
+          <Bar>
+            <BlueTxt>5 star</BlueTxt>
+            <Star5></Star5>
+            <BlueTxt>{this.state.star5}%</BlueTxt>
+          </Bar>
+          <Bar>
+            <BlueTxt>4 star</BlueTxt>
+            <Star4></Star4>
```
Comment (bshresthap) [inline client/src/components/ratingChart.jsx:109]: This is for displaying the bar chart ?

File: client/src/components/app.jsx [hunk 3]
```diff
@@ -84,6 +89,36 @@ class App extends React.Component {
       });
   }
 
+  getAnsweredQuestions(id) {
+    let productId = id;
+    axios.get(`http://localhost:4001/customer-questions/${productId}`)
```
Comment (bshresthap) [inline client/src/components/app.jsx:94]: Great job integrating the services!

File: client/src/components/itemColorOptions.jsx [hunk 4]
```diff
@@ -8,20 +7,13 @@ const ItemColorOptions = (props) => {
 
   const [color, setColor] = useState(props.color);
```
Comment (bshresthap) [inline client/src/components/itemColorOptions.jsx:8]: Great work using hooks!

**Review by bshresthap — COMMENTED** [review]
Great work Amos! Looks good.

File: client/src/components/itemColorOptions.jsx (hunk 1 above)
Comment (amosNash) [reply client/src/components/itemColorOptions.jsx:14]: Yeah I only have the same three color options for each item. So these colors correspond to a product.

File: client/src/components/ratingChart.jsx (hunk 2 above)
Comment (amosNash) [reply client/src/components/ratingChart.jsx:109]: Yes, I had a very tough time creating a chart from using a react chart library. I decided to create one myself so it is a bit hardcoded, but it was the solution I came to for getting a chart to popup.

**Pull request merged**